src/components/playback/playbackmanager.js

```diff
@@ -3324,7 +3322,13 @@ class PlaybackManager {
             if (errorOccurred) {
                 showPlaybackInfoErrorMessage(self, 'PlaybackError' + displayErrorCode);
             } else if (nextItem) {
-                self.nextTrack();
+                const apiClient = ServerConnections.getApiClient(nextItem.item.ServerId);
+
+                apiClient.getCurrentUser().then(function (user) {
+                    if (user.Configuration.EnableNextEpisodeAutoPlay) {
```

Comment:
Oh. Add here a check: if `nextItem.Type` is not a video, play next. Currently, music playback is broken.

Reply:
Fixed this. Had a sonarcloud failure as well so I refactored the function to prevent this failure. Hopefully makes it more readable now as well. I tested with movies, shows, music, photos, and home videos and it seems like there is no regression. Maybe give it a light once over to be sure.